docs/src/core/initial-setup.md

```diff
@@ -1,3 +1,68 @@
 # Initial Setup
 
+If you're interested in running Laniakea in Docker, see (this
+document)[https://github.com/lkorigin/laniakea/blob/master/tests/ci/Dockerfile]
```

Comment:
Oh, sorry that Dockerfile isn't for running Laniakea - it is for *building* it, so it may serve as inspiration for this document.
For running Laniakea in Docker (or some of its modules, rather) you would need to craft a very different Dockerfile.

(The Dockerfile there builds an environment for the CI to run the testsuite in, as Travis itself only has an ancient Ubuntu version and Laniakea requires at least Debian 10 (Buster)  to run)